### PR TITLE
zeroing PF clustering uncertainty

### PR DESCRIPTION
**Pull request by Sam-Harper**
#### PR description:

This PR zeros out the PF cluster uncertainty by default as discussed in the PPD general meeting last week. This because those values are not used anywhere, do not really make sense and will for the 2017UL be inaccurate in the endcap.

This is a super minor thing and should not hold anything. If it gets in it gets in, if it doesnt , it doesnt.

### CODE REVIEW

**Comment by perrotta** [comment]
please test

File: RecoParticleFlow/PFClusterProducer/plugins/CorrectedECALPFClusterProducer.cc [hunk 1]
```diff
@@ -131,6 +131,7 @@ void CorrectedECALPFClusterProducer::fillDescriptions(edm::ConfigurationDescript
     psd0.add<bool>("applyCrackCorrections", false);
     psd0.add<bool>("applyMVACorrections", false);
     psd0.add<bool>("srfAwareCorrection", false);
+    psd0.add<bool>("srfAwareCorrection",false);
```
Comment (perrotta) [inline RecoParticleFlow/PFClusterProducer/plugins/CorrectedECALPFClusterProducer.cc]: This was added twice, instead of "setEnergyUncertainty"

Comment (Sam-Harper) [reply RecoParticleFlow/PFClusterProducer/plugins/CorrectedECALPFClusterProducer.cc]: urgh, yes this came from the fact I had to rebase back to 11_X due to code formats :(

Comment (Sam-Harper) [reply RecoParticleFlow/PFClusterProducer/plugins/CorrectedECALPFClusterProducer.cc]: thanks fixed now

**Comment by perrotta** [comment]
please test

**Comment by Sam-Harper** [comment]
They were file IO errors. Which is good because this PR is really not important and it hadnt been a file IO error I would have simply just closed it and not bothered

**Comment by perrotta** [comment]
please test

**Comment by perrotta** [comment]
+1
- It zeroes out the PF cluster uncertainty by default, following a decision taken in the PPD general meeting
- Outputs show all correctedEnergyUncertanty'es zeroed, for all EGamma related clusters, as expected

@franzoni (+Phat, for whom I'm not able to find the github nickname...)

**Comment by fabiocos** [comment]
+1

**Pull request merged**